web/packages/shared/components/ToolTip/ToolTip.tsx

```diff
@@ -81,3 +84,8 @@ const StyledOnHover = styled(Text)`
   background-color: ${props => props.theme.colors.tooltip.background};
   max-width: 350px;
 `;
+
+const InfoIcon = styled(Icons.Info)`
+  font-size: 18px;
+  color: ${p => (p.muteIconColor ? p.theme.colors.text.disabled : 'inherit')};
```

Comment:
`muteIconColor` might be a good candidate for [a transient prop](https://styled-components.com/docs/api#transient-props).

OTOH, idk how to feel about `ToolTipInfo` having a `muteIconColor` prop while `InfoIcon` uses `$muteIconColor`. But in the end, the transient prop is for a reason - to avoid rendering that prop to the DOM.

Though to be honest I haven't even checked if the prop does indeed get rendered to the DOM.

Reply:
i tried to see if it does get rendered (in dev console from storybook) and didn't see the prop rendered. but regardless i thought it made sense to use it to mean `for styled component only`, so i revised